src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+
+public class Event extends Task {
+
+    protected String eventFrom;
+    protected String eventTo;
+
+    public Event(String description, String eventFrom, String eventTo) {
+        super(description);
+        this.eventFrom = eventFrom;
+        this.eventTo = eventTo;
+    }
+
+    public String toString() {
```

Comment:
Perhaps this method should not override the Object class method toString() since it does not necessarily print the Event object out. An alternative would be to use another method name that indicates a summary of the event.